### PR TITLE
Added tree schema, the human-friendly data catalog.

### PR DESCRIPTION
**Pull request by sewardgw**
Our product did not seem to fit into an existing category and, given the growth of metadata management and data catalogs as a whole I thought that the capabilities provided were unique enough to warrant placing into a new category. 

Happy to adjust this per feedback.

Tree Schema is free for teams up to 5 users, all accounts are opened in the free tier and no credit card is required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -436,6 +437,9 @@ Table of Contents
   * [synadia.com](https://synadia.com/ngs) — [NATS.io](https://nats.io) as a service. Global, AWS, GCP, and Azure. Free forever with 4k msg size, 50 active connections and 5GB of data per month.
   * [cloudkarafka.com](https://www.cloudkarafka.com/) - Free Shared Kafka cluster, up to 5 topics, 10MB data per topic and 28 days of data retention.
 
+## Metadata Management and Data Catalogs
+
+  * [Tree Schema](https://treeschema.com/) — Data catalog with APIs to manage data lineage as code. Free for teams of up to 5 users.
```
Comment (ripienaar) [inline README.md:442]: Please add this to an existing catagory, we dont do single entry catagories - you can also add other tools to this one if you prefer.

Comment (sewardgw-cf) [reply README.md]: I've moved Tree Schema to the Teams and Collaboration section.

Comment (sewardgw) [reply README.md]: I've moved Tree Schema to the Teams and Collaboration section.

**Comment by ripienaar** [comment]
thank you

**Pull request merged**